src/match_modality/README.md

```diff
@@ -49,15 +51,17 @@ A component that predicts which profiles from one modality might match with whic
 This component expects two h5ad files, `--input_mod1` and `--input_mod2`, for which the rows are shuffled and anonymised. These files have the following attributes:
 
   * `.X`: Sparse profile matrix.
+  * `.obs['group']`: Denotes whether a cell belongs to the 'train' or the 'test' set.
```

Comment:
maybe call this `.obs['is_train']` and store a boolean array? That would be consistent with open problems then.

Reply:
This would need to be updated across tasks too. I don't really have a preference here, but making components easier to integrate with open problems is definitely preferred.

Reply:
For the sake of record keeping, in a meeting on Friday we decided to move the train and the test data to separate files because this would allow people to pre-train a model and apply it to the test data without having to read in the train data.

I will adapt the API for task 2. Unfortunately, this means a component will need to read in 5 different files, I believe.